src/coreclr/nativeaot/System.Private.CoreLib/src/System/Reflection/Assembly.NativeAot.cs

```diff
@@ -35,6 +35,15 @@ public static Assembly Load(string assemblyString)
             return Load(name);
         }
 
+        // Performance metric to count the number of assemblies
+        // Caching since in NativeAOT, the number will be the same
+        private static uint? s_assemblyCount;
+        internal static uint GetAssemblyCount()
+        {
+            s_assemblyCount ??= (uint)AppDomain.CurrentDomain.GetAssemblies().Length;
```

Comment:
What happened here? It looks like this was fixed, then reverted

Reply:
Sigh.. didn't sync correctly. Thanks for the catch!